web-common/src/features/dashboards/time-series/MeasureChart.svelte

```diff
@@ -76,7 +77,7 @@
   $: numberKind = numberKindForMeasure(measure);
 
   const tweenProps = { duration: 400, easing: cubicOut };
-  const xScale = getContext(contexts.scale("x")) as ScaleStore;
+  const xScale = getContext(contexts.scale("x"));
```

Comment:
**Issue**: The getContext function is used without type assertion, which may lead to runtime errors if the context does not match the expected type. <br> **Fix**: Reinstate the type assertion for getContext to ensure type safety. <br> **Code Suggestion**: 
 ```
 -  const xScale = getContext(contexts.scale("x"));
 +  const xScale = getContext(contexts.scale("x")) as ScaleStore;
 ```